src/Tokenizers/CSS.php

```diff
@@ -510,7 +510,6 @@ public function tokenize($string)
             if ($finalTokens[($numTokens - 2)]['content'] === '') {
                 unset($finalTokens[($numTokens - 2)]);
                 $finalTokens = array_values($finalTokens);
-                $numTokens   = count($finalTokens);
```

Comment:
From the code flow it is cleat that the variable is not used because there is a return statement at the end of the code block. But I don't mind, we can revert this change.